discovery-provider/plugins/notifications/src/processNotifications/mappers/usdcWithdrawal.ts

```diff
@@ -0,0 +1,53 @@
+import { Knex } from 'knex'
+import { NotificationRow } from '../../types/dn'
+import { USDCWithdrawalNotification } from '../../types/notifications'
+import { BaseNotification } from './base'
+import { logger } from './../../logger'
+import { sendTransactionalEmail } from '../../email/notifications/sendEmail'
+import { buildUserNotificationSettings } from './userNotificationSettings'
+import { email } from '../../email/notifications/preRendered/withdrawal'
+
+type USDCWithdrawalRow = Omit<NotificationRow, 'data'> & {
+  data: USDCWithdrawalNotification
+}
+
+export class USDCWithdrawal extends BaseNotification<USDCWithdrawalRow> {
+  userId: number
+  amount: number
+  receiverAccount: string
+  signature: string
+
+  constructor(dnDB: Knex, identityDB: Knex, notification: USDCWithdrawalRow) {
+    super(dnDB, identityDB, notification)
+    const userIds: number[] = this.notification.user_ids!
```

Comment:
can you check and throw a specific error here? i assume this is meant to never be empty but in the case of a bug throwing a specific error with some data would likely help a lot

Reply:
👍